### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by VincentLim** [doublespeakgames/adarkroom#87]
Hi, 
First of all, it's a nice game :), thanks.

And the issue, as far as i am (coal mine) :

In the top right panel in the forest, the 'iron mine' and the 'coal mine' are not translated in french (respectively: 'mine de fer', 'mine de charbon')

Vincent.

PS : I add some 
sulphur mine => mine de soufre
not enough sulphur => soufre insuffisant.
sulphur => soufre
bayonet => baïonnette

In French, souffre is mispelled => soufre.

**Comment by alexeiberkov** [comment]
Hi.
Very interesting and nice game! I am empressed. 
There is not enough translations for russian localisation (lang/ru/strings.po). Is it okay to make a patch as well ?

**Comment by Continuities** [comment]
If you've found a problem with the RU translations, please do help!

**Comment by GSimas** [comment]
Brazilian portuguese translation is improving
and it should work...
(the link is right <http://adarkroom.doublespeakgames.com/?lang=pt_br> but then you go... is english)

**Comment by grotewold** [comment]
Yeah. We haven't made a push to the production server in a while. We'll do that here soon. 




Cc: @Continuities

**Comment by gitupofftheflooranddosomework** [comment]
This is completed a while ago and can be closed
